src/main/scala/net/snowflake/spark/snowflake/SnowflakeConnectorUtils.scala

```diff
@@ -34,7 +34,7 @@ object SnowflakeConnectorUtils {
     * Check Spark version, if Spark version matches SUPPORT_SPARK_VERSION enable PushDown,
     * otherwise disable it.
     */
-  val SUPPORT_SPARK_VERSION = "3.2"
+  val SUPPORT_SPARK_VERSION = "3.3"
```

Comment:
This is the only production code change. All the others are test related changes.